docs/modules/ROOT/pages/build.adoc

```diff
@@ -0,0 +1,62 @@
+= Build
+:meta-updater-github-url: https://github.com/advancedtelematic/meta-updater/tree/master
+
+== Quickstart
+
+If you don't already have a Yocto project that you want to add OTA to, you can use the xref:dev@getstarted::raspberry-pi.adoc[HERE OTA Connect Quickstart] project to rapidly get up and running on a Raspberry Pi. It takes a standard https://www.yoctoproject.org/tools-resources/projects/poky[poky] distribution, and adds OTA and OSTree capabilities.
+
+== Dependencies
+
+//MC: TOMERGE: These "dependencies" mostly just duplicates the prerequisite sections: https://main.gitlab.in.here.com/olp/edge/ota/documentation/ota-connect-docs/blob/master/docs/getstarted/modules/ROOT/pages/raspberry-pi.adoc
```

Comment:
Yes, but the target audience is slightly different. This assumes that the user already has a Yocto environment, so this specifically calls out only the extra things that we need in addition to the default Yocto dependencies.

That said, this may or may not be useful, so I'm not saying it has to stay the way it is.